### PR TITLE
feat: add remove job dialog

### PR DESCRIPTION
**Pull request by collinlokken**
## What does this pull request change?
Adds popup for confirmation of job cancelation

## Why is this pull request needed?
Requested feature

## Issues related to this change
https://github.com/equinor/dm-job/pull/128

### CODE REVIEW

**Comment by collinlokken** [comment]
image

**Comment by collinlokken** [comment]
Uploading Screen Recording 2023-11-28 at 16.19.29.mov…

**Comment by collinlokken** [comment]
https://github.com/equinor/dm-core-packages/assets/43639886/fdffa322-4c43-4ba0-8dab-43019c4dd39e

File: packages/dm-core-plugins/src/job/RemoveJobDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  isOpen: boolean
+  remove: () => void
+  afterRemove: () => void
+}
+export const RemoveJobDialog = (props: RemoveJobDialogProps) => {
+  const { isOpen, remove, afterRemove } = props
+  return (
+    <Dialog open={isOpen}>
+      <Dialog.CustomContent>
+        <pre> Are you sure you want to stop the current job? </pre>
```
Comment (soofstad) [inline packages/dm-core-plugins/src/job/RemoveJobDialog.tsx:15]: \<pre\> is not the preferred html element for standard text...
Please change to "Typography" or \<p\>

**Review by soofstad — CHANGES_REQUESTED** [review]

File: packages/dm-core-plugins/src/job/RemoveJobDialog.tsx (hunk 1 above)
Comment (collinlokken) [reply packages/dm-core-plugins/src/job/RemoveJobDialog.tsx:15]: image

Comment (collinlokken) [reply packages/dm-core-plugins/src/job/RemoveJobDialog.tsx]: image

**Review by soofstad — APPROVED** [review]

**Comment by pbullhove** [comment]
image
How about center the buttons like this?

**Comment by collinlokken** [comment]
image

**Pull request merged**